### DISCUSSION TITLE
[@types/chrome] optional_host_permissions attribute is missing

### DISCUSSION THREAD

**Discussion by aspiers** [DefinitelyTyped/DefinitelyTyped#68051] [Issues with a @types package]
[`chrome.runtime.ManifestV3`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/de090eedb92ff8fda14f7ff7f245df0c61dc5015/types/chrome/index.d.ts#L7430) correctly contains the following types relating to permissions:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/de090eedb92ff8fda14f7ff7f245df0c61dc5015/types/chrome/index.d.ts#L7460-L7462

However it is missing `optional_host_permissions` which is clearly documented at https://developer.chrome.com/docs/extensions/develop/concepts/declare-permissions

I believe the correct type is `string[] | undefined` since it's the same format as `host_permissions`.

[a comment by an automated account is not shown]

**Comment by fregante** [comment]
- Done: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68210
